src/client/cypress/e2e/detailPage/sideNavigation.cy.js

```diff
@@ -0,0 +1,131 @@
+import { addItem, saveForm } from "../helpers/buttonHelpers.js";
+import { clickOnRowWithText, showTableAndWaitForData } from "../helpers/dataGridHelpers.js";
+import { setInput, setSelect } from "../helpers/formHelpers.js";
+import {
+  createBorehole,
+  createFieldMeasurement,
+  createGroundwaterLevelMeasurement,
+  createHydrotest,
+  createWateringress,
+  goToRouteAndAcceptTerms,
+  returnToOverview,
+  startBoreholeEditing,
+} from "../helpers/testHelpers";
+
+describe("Test for the detail page side navigation.", () => {
+  it("tests if navigation points are greyed out if there is no content", () => {
+    // Create a borehole and store its ID
+    createBorehole({ "extended.original_name": "AAA_HIPPOPOTHAMUS", "custom.alternate_name": "AAA_HIPPOPOTHAMUS" }).as(
+      "borehole_id",
+    );
+
+    cy.get("@borehole_id").then(id => {
+      goToRouteAndAcceptTerms(`/${id}`);
+      startBoreholeEditing();
+    });
+
+    // Check that some menu items are enabled (not greyed out)
+    cy.get('[data-cy="location-menu-item"]').should("have.css", "color", "rgb(153, 25, 30)");
+    cy.get('[data-cy="borehole-menu-item"]').should("have.css", "color", "rgba(0, 0, 0, 0.87)");
+    cy.get('[data-cy="status-menu-item"]').should("have.css", "color", "rgba(0, 0, 0, 0.87)");
```

Comment:
PP: checks in function auslagen und wiederverwenden.
  
  Irgendwie so, oder eine function für beide 🤷‍♀️ .
  
  const isInactive = menuItem => {
    getElementByDataCy(menuItem).should("have.css", "color", "rgba(0, 0, 0, 0.87)");
  };